mkdocs/sys-amin/docs/install-tactic-centos7.md

```diff
@@ -0,0 +1,152 @@
+# Install TACTIC on CentOS 7
+
+>Updated December 2019
+
+These instructions are for a TACTIC server that will use PostgreSQL and Apache on a Cent OS server. Refer to the General TACTIC Installation for other OSes. For other database types, refer to guides in Install TACTIC Application.
+
+
+## Install PostgreSQL
+
+CentOS 7 comes with PostgreSQL 9.2 by default. TACTIC needs PostgreSQL 9.4 and higher.
+
+Here are instructions to install PostgreSQL 11 using official PostgreSQL packages.
+
+    rpm -Uvh https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+    yum update
+    yum install postgresql11-server postgresql11-contrib postgresql11-devel
+    /usr/pgsql-11/bin/postgresql-11-setup initdb
+
+
+
+### Starting PostgreSQL and enabling the service
+
+    systemctl start postgresql-11
+    systemctl enable postgresql-11
+
+
+## Configure PostgreSQL
```

Comment:
Since the user hasn't installed the RPM yet, they won't have the source code, and the pg_hba.conf file. 
Do you think this section should be moved to after the RPM is installed like I did in the RHEL docs?